### DISCUSSION TITLE
Move new command row to the bottom of DataGrid?

### DISCUSSION THREAD

**Discussion by Bassist067** [Megabit/Blazorise#2940] [Q&A]
Is it possible to move new command row with the command button? I was able to move the button only.

![image](https://user-images.githubusercontent.com/19996164/134005385-005b86a1-2ac8-493f-8934-115fae62dfe2.png)

**Comment by David-Moreira** [comment]
Hello,
I am not sure I follow.
Would you like to place the New / Edit / Remove buttons outside of the Grid?
Check our Button Row example at: https://bootstrapdemo.blazorise.com/tests/datagrid
(Change CommandMode to ButtonRow)

Is this what you are looking for?
![image](https://user-images.githubusercontent.com/22283161/134052396-1f40ece0-caf1-40a5-b8b3-b483bdb0c515.png)

Also New/Edit/Remove API is now publicly available so you can just grab a reference to the Datagrid and call them whenever you want. So your free to allocate them anywhere

**Reply by Bassist067** [reply]
The button row allows me to render buttons outside of the grid, which is nice. Please note I'm talking about the new item row itself. It's always appearing at the top of the other rows.

Is it possible to render the new row below the other rows?

**Reply by David-Moreira** [reply]
Hello,
Take a look at the `PagerPosition` Parameter. Since the ButtonRow is allocated in the Pager, you will have to set the `PagerPosition` to Bottom.
